test/org/sagebionetworks/bridge/dynamodb/DynamoSchedulePlanTest.java

```diff
@@ -51,6 +61,9 @@ public void canSerializeDynamoSchedulePlan() throws Exception {
         assertEquals(plan.getGuid(), plan2.getGuid());
         assertEquals(plan.getLabel(), plan2.getLabel());
         assertEquals(plan.getModifiedOn(), plan2.getModifiedOn());
+        
+        ScheduleStrategy retrievedStrategy = plan.getStrategy();
+        assertEquals(retrievedStrategy, strategy);
```

Comment:
Minor: Do we want to assert that study key is null? Also, do we want to write a study key into the JSON to verify that fromJson() ignores it?